src/Version.php

```diff
@@ -32,11 +32,56 @@ public static function isStable()
     }
 
     /**
-     * Compares a version to Bolt's version.
+     * Compares a semantic version against Bolt's version, given a specified
+     * comparison operator.
      *
-     * Note: Be sure to include the `.z` number in the version given, as omitting it can give inconsistent results.
-     * For example, if the current version is `3.3.0`, `compare('3.3', '>=')` returns false. In reality you want
-     * the check to return true for this case, and `compare('3.3.0', '>=')` _does_ return true.
+     * @see http://semver.org/ For an exmplaination on semantic versioning.
+     *
+     * Note: Be sure to include the `.z` number in the version given, as
+     * omitting it can give inconsistent results. See below at the results in
+     * Tables 1 & Table 2 for more detail.
+     *
+     * Note: Depending on the stability of the VERSION constant, the output of
+     * this may give, initially, unexpected results.
+     *
+     * The two tables below show `Version::compare() calls, compared version,
+     * operator, and their result. Firstly versus Bolt 3.2.0, a stable version
+     * (Table 1). Secondly against Bolt 3.2.0-beta9, a beta (Table 2).
+     *
+     * Table 1: `const VERSION = '3.2.0'
+     * +------------------+----+--------+
+     * | Compared Version |    | Result |
+     * +------------------+----+--------+
+     * | 3.1              | >= | false  |
+     * | 3.1.0            | >= | false  |
+     * | 3.2              | >= | false  |
+     * | 3.2.0-beta8      | >= | false  |
+     * | 3.2.0-beta9      | >= | false  |
+     * | 3.2.0            | >= | true   |
+     * | 3.2.1            | >= | true   |
+     * | 3.3              | >= | true   |
+     * | 3.3.0            | >= | true   |
+     * +------------------+----+--------+
+     *
+     * Table 2: `const VERSION = '3.2.0 beta 9'`
+     * +------------------+----+--------+
+     * | Compared Version |    | Result |
+     * +------------------+----+--------+
+     * | 3.1              | >= | false  |
+     * | 3.1.0            | >= | false  |
+     * | 3.2              | >= | false  |
+     * | 3.2.0-beta8      | >= | false  |
+     * | 3.2.0-beta9      | >= | true   |
+     * | 3.2.0            | >= | true   |
+     * | 3.2.1            | >= | true   |
+     * | 3.3              | >= | true   |
+     * | 3.3.0            | >= | true   |
+     * +------------------+----+--------+
+     *
+     * Note that pre-release versions, such as 3.2.0-beta, are considered lower
+     * than their final release counterparts (like 2.3.0).  As you may notice,
```

Comment:
`like 2.3.0` or `like 3.2.0`?

Two spaces after the period. You old-fashioned, you.